recipes/python-rnacentral/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: python-rnacentral
+  version: "1.0.0"
+
+about:
+  home: https://github.com/fabio-cumbo/RNAcentral-data-retrieval-tool
+  summary: 'This tool exploits the RNAcentral public APIs to
+  automatically retrieve sequences (converted to the FASTA format)
+  and the related metadata, starting with one or a list of
+  RNAcentral IDs'
+  license: BSD
```

Comment:
do you have a license file? license_file
And license_family could also be added.